### PR TITLE
Point to ACK documentation in the main README

### PR DESCRIPTION
**Pull request by akartsky**
Remove redundant instructions and point to ACK documentation in the main README

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by surajkota — APPROVED** [review]
/approve
/lgtm

File: README.md [hunk 1]
```diff
@@ -35,371 +35,33 @@ Find the helm charts and controller images on Amazon ECR Public Gallery.
...
-  ]
-}
-' > ./pass_role_policy.json
-```
-
-Run the following command to create a role with the trust relationship defined in `trust.json`. This role enables the Amazon EKS cluster to get and refresh credentials from IAM.
-
-```sh
-OIDC_ROLE_NAME=ack-controller-role-$CLUSTER_NAME
+### 3.0 Samples
```
Comment (mbaijal) [inline README.md:46]: Does this need to be a separate section ? Maybe we can just combine these with the individual installation sections ?

File: README.md [hunk 2]
```diff
@@ -35,371 +35,33 @@ Find the helm charts and controller images on Amazon ECR Public Gallery.
...
-  trainingJobName: ack-sample-tainingjob
-  roleARN: <sagemaker_execution_role_arn>
-  ...
-```
+Head over to the [Manage Resources In Multiple AWS Accounts](https://aws-controllers-k8s.github.io/community/docs/user-docs/cross-account-resource-management/)
 
-### 8.0 Adopt Resources
+### 6.0 Adopt Resources
 
 ACK controller provides to provide the ability to “adopt” resources that were not originally created by ACK service controller. Given the user configures the controller with permissions which has access to existing resource, the controller will be able to determine the current specification and status of the AWS resource and reconcile said resource as if the ACK controller had originally created it.
```
Comment (mbaijal) [inline README.md:66]: Link to official doc similar to cross account ?

File: README.md (hunk 1 above)
Comment (mbaijal) [reply README.md:46]: ok for now.

**Pull request merged**